code/modules/projectiles/guns/projectile/revolver.dm

```diff
@@ -107,13 +107,13 @@
 	name = "\"The Judge\""
 	desc = "A revolving hand-shotgun by Cybersun Industries that packs the power of a 12 guage in the palm of your hand (if you don't break your wrist). Uses 12 shotgun rounds."

```

Comment:
I completely overlooked this. I should fix this later.

"Uses 12 shotgun rounds." Uh, no. Who's the idiot who wrote this? I should give him a piece of my mind.

... OH, wait, _I_ wrote this. Well this is awkward.